src/app/backup/list/restore/template.html

```diff
@@ -99,8 +99,7 @@
                 <button mat-icon-button
                         [attr.id]="'km-delete-restore-' + backup.name"
                         matTooltip="Delete Automatic Backup"
-                        kmThrottleClick
-                        (throttleClick)="delete(backup); $event.stopPropagation()"
+                        (click)="delete(backup); $event.stopPropagation()"
```

Comment:
It was not required as API is not called in the handler. It opens a dialog which doesn't need additional throttling.